src/utils/navbarChangeHandlers.ts

```diff
@@ -0,0 +1,13 @@
+export const handleNavbarScroll = () => {
+  const navbar = document.querySelector('.navbar');
+  if (navbar) {
+    navbar.classList.toggle('scroll', window.scrollY > window.innerHeight - 60);
+  }
+};
+
+export const handleNavbarBackground = () => {
+  const navbar = document.querySelector('.navbar');
+  if (navbar) {
+    navbar.classList.toggle('homepage-navbar');
```

Comment:
Shouldn't be like this?

```suggestion
    navbar.classList.add('homepage-navbar');
```

Reply:
It works fine with `add`, we could even change `toggle` that we have above, in `handleNavbarScroll` function into `add`. 
So instead 
`navbar.classList.toggle('scroll', window.scrollY > window.innerHeight - 60);`
we could have 
`window.scrollY > window.innerHeight - 60 && navbar.classList.add('scroll');`
It's a little but weird for me that it works fine in case of `scroll` - I scroll on homepage so the class was added and why it is not there after I navigate for example to the `/docs` 🤷‍♀️

Reply:
When I am thinking super hard about it maybe it has some sense, but it's one of those tricky things to understand for my mind 🙈